visits-service-new/src/main/java/com/petclinic/visits/visitsservicenew/PresentationLayer/VisitController.java

```diff
@@ -15,6 +15,14 @@
 public class VisitController {
     private final VisitService visitService;
 
+    @GetMapping(value="", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+    public Flux<VisitResponseDTO> getAllVisits(){
+        return visitService.getAllVisits();
+    }
+    @GetMapping(value="/pets/{petId}", produces = MediaType.TEXT_EVENT_STREAM_VALUE)
+    public Flux<VisitResponseDTO> getVisitsForPet(@PathVariable int petId){
+        return visitService.getVisitsForPet(petId);
+    }
```

Comment:
Methods are reactive, could return Flux<ResponseEntity<VisitResponseDTO>> to check status code and handle exceptions, both methods look fine.